sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -3803,8 +3803,10 @@ object GpuOverrides extends Logging {
           TypeSig.ARRAY + TypeSig.DECIMAL_128 + TypeSig.BINARY +
           GpuTypeShims.additionalCommonOperatorSupportedTypes).nested(), TypeSig.all),
       (filter, conf, p, r) => new SparkPlanMeta[FilterExec](filter, conf, p, r) {
-        override def convertToGpu(): GpuExec =
-          GpuFilterExec(childExprs.head.convertToGpu(), childPlans.head.convertIfNeeded())
+        override def convertToGpu(): GpuExec = {
+          GpuFilterExec(childExprs.head.convertToGpu(),
+            childPlans.head.convertIfNeeded())(conf.isTieredProjectEnabled)
```

Comment:
(nit) Should we use the named parameter `useTieredProject` here, and in the few other cases where we pass it, and not rely on the order of the two boolean paramaters?